client/stt.py

```diff
@@ -277,7 +277,8 @@ class GoogleSTT(AbstractSTTEngine):
 
     To obtain an API key:
     1. Join the Chromium Dev group:
-       https://groups.google.com/a/chromium.org/forum/?fromgroups#!forum/chromium-dev
+       https://groups.google.com/a/chromium.org/forum/?fromgroups#!forum/
```

Comment:
Exactly, i don't remember why is here :)
I believe i do that to respect quantity of letters on one line